### PR TITLE
Fix ScrollContainer touch-scrolling not working.

### LINKED ISSUE #65992: ScrollContainer touch input not working on android
### Godot version

4.0.beta1

### System information

Android 10 - Samsung Galaxy Tab S4 / Android 13 - One Plus 10 Pro

### Issue description

Using the scroll container on android, seems to ignore the touch input on the main scrollable content. Using the ScrollBar's at the bottom / left works, but is not really user-friendly on such small devices.

At the moment, it seems to be impossible to use touch input on the scrollable content, as nothing happens. I would expect to be able to scroll on the main content with my finger, which should move the scrollable content. 

The same behaviour can also be reproduced with the following setting enabled `input_devices/pointing/emulate_touch_from_mouse`. With this setting enabled in 3.5 you are able to emulate the panning on your desktop, that you would normally do on your phone. But this also does not seem to be working in 4.0.

My expected behavior fully aligns with what happens in the 3.5 reproduction.

Am I doing something wrong in my 4.0 scene? Or might this be a regression?

### Steps to reproduce

I have the following scene set up:
image

The mouse filter of the texture rect is set to PASS, same goes for the scroll container, as this is the default value when adding to the scene.

I have also reproduced the exact same scene in 3.5, which works flawlessly.

### Minimal reproduction project

For 4.0: [ScrollContainer_touch_input_reproduction_v4.0.zip](https://github.com/godotengine/godot/files/9592048/ScrollContainer_touch_input_reproduction_v4.0.zip)

For 3.5 (was built with the mono version, don't know if that matters): [ScrollContainer_touch_input_reproduction_v3.5.zip](https://github.com/godotengine/godot/files/9592057/ScrollContainer_touch_input_reproduction_v3.5.zip)

### LINKED ISSUE #66554: ScrollContainer touch-scrolling not works.
### Godot version

master (14e1f36 on 20220928) 

### System information

Windows 11

### Issue description

In 3.x we can touch to scroll ScrollContainer, and on Windows it need to enable `input_devices/pointing/emulate_touch_from_mouse`.
Now it doesn't work in 4.0. 

### Steps to reproduce

N/A

### Minimal reproduction project

[scroll_bug.zip](https://github.com/godotengine/godot/files/9666664/scroll_bug.zip)

### PR DESCRIPTION
**Pull request by zaevi**
Fixes #66554 and fixes #65992.

### CODE REVIEW

File: platform/windows/display_server_windows.cpp [hunk 1]
```diff
@@ -565,7 +565,7 @@ bool DisplayServerWindows::screen_is_touchscreen(int p_screen) const {
 #ifndef _MSC_VER
 #warning touchscreen not working
 #endif
-	return false;
+	return DisplayServer::screen_is_touchscreen(p_screen);
```
[the remark this answers is not in the document]
Comment (zaevi) [reply platform/windows/display_server_windows.cpp:568]: Yes, the behavior should be similar to `OS::has_touchscreen_ui_hint()` in 3.x. Windows doesn't support touchscreen yet but it should allow emulating from mouse.

**Comment by zaevi** [comment]
This PR would fix both `Touch emulating not works on Windows` and `ScrollContainer touch-scrolling not works (for all platform)`

**Comment by qq715152910** [comment]
Has this PR been merged？ I'm waiting for the fixed version as I'm having the same problem.

**Comment by akien-mga** [comment]
This PR is 12 hours old, please be patient.

**Comment by qq715152910** [comment]
This PR doesn't work on Android anymore.

**Comment by qq715152910** [comment]
This PR does not work on Android, test version 4.0-Beta2

**Comment by zaevi** [comment]
> This PR does not work on Android, test version 4.0-Beta2

This PR isn't merged yet. Do you build for Android in local using this branch?

**Comment by qq715152910** [comment]
> 

Yes, I modified the source code according to your PR and rebuilt it. Tested in 4.0-Beta1, it works well on Windows, and works well on Android.
When rebuilt under 4.0-Beta2 source code, Windows also worked fine, but when tested on Android, it no longer worked.

**Comment by qq715152910** [comment]
> 

Yes, I modified the source code according to your PR and rebuilt it. Tested in 4.0-Beta1, it works well on Windows, and works well on Android.
When rebuilt under 4.0-Beta2 source code, Windows also worked fine, but when tested on Android, it no longer worked

**Comment by zaevi** [comment]
> > 
> 
> Yes, I modified the source code according to your PR and rebuilt it. Tested in 4.0-Beta1, it works well on Windows, and works well on Android. When rebuilt under 4.0-Beta2 source code, Windows also worked fine, but when tested on Android, it no longer worked

I just compiled based on c477e7c (with this fix merged) and it works as expected on Android.

You can retry using my reproduction project in #66554.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**